bin/convert_to_gcmt

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+
+"""
+A Command line script that creates k-tables for exorad
+"""
+import os
+import argparse
+import yaml
+
+from GCMtools import GCMT
+
+wdir = os.getcwd()
+
+################
+# Default values
+################
+DEFAULT_CONFIG = os.path.join(wdir, 'convert.yaml')
+DEFAUT_PREFIXES = ["T","U","V","W"]
+DEFAULT_ITERATIONS = "all"
+DEFAULT_LOAD_EXISTING = False
+DEFAULT_GCM = "MITgcm"
+DEFAULT_TAG = "nc_convert"
+DEFAULT_DATA_PATH = os.path.join(wdir, 'run')
+DEFAULT_SAVE_PATH = os.path.join(wdir, 'results')
+
+########################
+# Command line arguments
+########################
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-c",
+    "--config",
+    help="specify the path to the config file that holds the config for converting to xarray",
+    default=DEFAULT_CONFIG,
+)
+args = parser.parse_args()
+
+###################################
+# Load config
+###################################
+if not os.path.isfile(args.config):
+    print("no config file found, using defaults instead")
```

Comment:
Not really important, but we could think about using the wrt class here as well for more uniform style.

Reply:
I agree. I will change that.